tools/tasks/gulp-pa11y.js

```diff
@@ -0,0 +1,92 @@
+const fs = require('fs');
```

Comment:
Should this file be considered for a standalone library?

Reply:
I need clarification here.

Reply:
E.g., github.com/phase2/gulp-pa11y

Reply:
Ah, we're talking about the whole file (thought we were talking about `fs`).

So, bringing gulp-pa11y.js, gulp-sass2json.js, and gulp-twig-namespaces.js to some kind of individual package would be great, BUT these things absolutely need to live as close to this project as possible.

Monorepo time? Lerna? For the time being, I'm OK with these just hanging out in `tools/tasks` since I consider the Pattern Lab Starter as much a node app as a Drupal theme.

Reply:
I don't see this as a monorepo thing, I just see this as being clearly not all that tied to PL and possibly of use to other projects, so shouldn't it be it's own Open Source project with an issue queue that PLS depends on? Admittedly, I don't see whatever needs it to be kept close to PLS and it's not a blocking point.

Reply:
One of the big goals of moving to Webpack and folding in p2-theme-core into PLSv10 was to keep all the tooling as close at hand as possible. And, honestly, it would more likely need to be a PR against a project like this instead: https://github.com/dreamzmaster/gulp-pa11y

Let's give it some time out in the world to see how it improves.

Reply:
just seconding that i don't think it's out of place here, doubly so considering all of the "accessibility best practice" default patterns we want to fold in, this seems like a logical extension to include in the pls toolkit